themes/default/content/docs/using-pulumi/adopting-pulumi/import/_index.md

```diff
@@ -12,161 +12,184 @@ aliases:
 - /docs/guides/adopting/import/
 ---
 
-Most infrastructure projects require working with existing cloud resources, either by building on top of existing
-resources or adopting existing resources under management with a new and more robust infrastructure provisioning solution.
+Most infrastructure as code projects require working with existing cloud resources, whether those resources were originally created with another IaC tool or manually provisioned with a cloud provider console or CLI. Interacting with a previously created cloud resource with Pulumi typically happens in one of two ways:
 
-No matter how you've provisioned these resources &mdash; manually in your cloud provider's console or CLI, using an
-infrastructure as code tool like Terraform or AWS CloudFormation &mdash; Pulumi enables you to adopt and manage your resources.
+* Referencing the properties of the existing cloud resource in order to use those properties to configure a Pulumi-managed resource.
```

Comment:
Debatable, and nit-picky, but because it says "one of two ways", I would use an ordered list. Alternatively, could word the preview paragraph as "one of the following ways".

```suggestion
1. Referencing the properties of the existing cloud resource in order to use those properties to configure a Pulumi-managed resource.
```